qeynos2/Rineval_Talyas.lua

```diff
@@ -1,11 +1,12 @@
 -- items: 19071, 19070, 1000, 18302
 function event_say(e)
 	if(e.message:findi("hail")) then
-		e.self:Say(string.format("hello, %s",e.other:GetName()));
+		e.self:Say(string.format("hello, " .. e.other:GetName()));
 	elseif(e.message:findi("scribe")) then
-		e.self:Emote(string.format("looks at %s strangely. 'Who told you this?'",e.other:GetName()));
-	elseif(e.message:findi("Karn")) then
-		e.self:Say("'I see. If a scribe is what you seek, then bring me the torn books of fire and ice. A donation of 1000 platinum is needed as well. The donation will be offered as a blessing to the gods.");
+		e.self:Emote(string.format("looks at " .. e.other:GetName() .. " strangely. 'Who told you this?'"));
```

Comment:
Same here, string.format is a valid method, but if you plan on changing, to not use it, please remove the string.format call.

Reply:
Thanks, fixed